src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -284,8 +289,10 @@ class PillSelectorBase extends React.Component<Props, State> {
                         {...rest}
                         {...inputProps}
                         autoComplete="off"
-                        className={classNames('bdl-PillSelector-input', 'pill-selector-input', className)}
-                        disabled={disabled}
+                        className={classNames('bdl-PillSelector-input', 'pill-selector-input', className, {
+                            'input-in-new-line': inputInNewLine,
```

Comment:
new classnames should follow SUIT naming:

https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md

this could be a modifier:
bdl-PillSelector-input--nextLine